### DISCUSSION TITLE
imswel_person card wrong language

### DISCUSSION THREAD

**Discussion by ChrissHF** [UI-Lovelace-Minimalist/UI#622] [Q&A]
Hi,
For some reason the text for find my device in the custom_card_imswel_person is in Portuguese.
My Dashboard and HA language is set to German.
What do I need to adjust to change it?

UPDATE: Figured it out myself, had to move the language file to the "root" directory of the custom card

**Comment by mdb17** [comment]
I am getting an error saying it cant find the language template when adding the card. Do you have an suggestions on how to start troubleshooting it?
